EAD/Session 4/Assignment4/src/controller/AddEmployeeServlet.java

```diff
@@ -0,0 +1,81 @@
+package controller;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import dto.EmployeeDTO;
+import facade.EmployeeFacade;
+import util.Validation;
+
+/**
+ * 
+ * @author Nilesh Patel
+ * This servlet add employee
+ */
+@WebServlet("/AddEmployeeServlet")
+public class AddEmployeeServlet extends HttpServlet 
+{
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * @see HttpServlet#HttpServlet()
+	 */
+	public AddEmployeeServlet() 
+	{
+		super();	
+	}
+
+	/**
+	 * This method add employee
+	 * @param request object of HttpServletRequest
+	 * @param response object of HttpServletResponse
+	 */
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException 
+	{
+
+		EmployeeDTO employee = new EmployeeDTO();
+		HttpSession session = request.getSession();
+		try 
+		{
+			employee.setName(request.getParameter("name"));
+			employee.setGender(request.getParameter("gender"));
+			employee.setEmail(request.getParameter("email"));
+			employee.setPassword(request.getParameter("password"));
+			String confirmPassword = request.getParameter("confirmpassword");
+			employee.setPhoneNumber(request.getParameter("contactnumber"));
+			employee.setOrganization(request.getParameter("organization"));

```

Comment:
If there is no organisation selected by default what will be the value. Handle the case

Reply:
Default will be MetaCube.